src/OlWebsiteApp.jsx

```diff
@@ -23,7 +23,7 @@ export default function OlWebsiteApp() {
           <ScrollToTop>
             <div className='app-body'>
               <Switch>
-                <Route exact path='/contact-us' component={ContactUsView} />
+                <Route exact path='/contact' component={ContactUsView} />
```

Comment:
I believe we can get rid of the exact on the contact route.

Reply:
Low priority to block a merge i guess but i agree